docs/JupyterNotebook.md

```diff
@@ -1,8 +1,80 @@
 # Talk2PowerSystem Jupyter Notebook
 
-##  Run the notebook
+## Run the chatbot on your developer machine using a Jupyter notebook
 
 ```bash
+conda create --name Talk2PowerSystemLLM --file conda-linux-64.lock
 conda activate Talk2PowerSystemLLM
+poetry install
 jupyter notebook
 ```
+
+The notebook will prompt you for the GraphDB and Azure OpenAI credentials.
+  - Graphdb password is shared in Graphwise Keeper, entry `DATA/CIM/CIM GraphDB`
+  - Azure OpenAI credentials are shared in Graphwise Keeper, entry `Statnett/Graphwise Azure Statnett Deployment GPT4.1`
+
+## Run the chatbot on the RNDP environment using a Jupyter notebook
+
+- Open https://rndp.statnett.no/ from your browser.
+- Login with you Statnett Microsoft Account.
+- A screen like this should appear:
+![server_options_screen.png](images/server_options_screen.png)
+- For size pickup `Small` and click `Start`
+- It takes a while to start the server
+ ![server_starting.png](images/server_starting.png)
+- After that you should see a screen like this:
+![landing.png](images/landing.png)
+- From the left menu select git:
+![git.png](images/git.png)
+- Click the blue button `Clone a Repository`, enter https://github.com/statnett/Talk2PowerSystem_LLM for URI and click `Clone`:
+![clone.png](images/clone.png)
+- A folder named `Talk2PowerSystem_LLM` should appear under your home directory `/home/{your-user}`:
+![repository.png](images/repository.png)
+- From the `Launcher` tab section `Other` open a terminal
+- A screen like this should appear:
+![terminal.png](images/terminal.png)
+- Check your working directory:
+```commandline
+pwd
+```
+It should be your home directory.
+- Setup the environment:
+  - Execute:
+    ```commandline
+    cd Talk2PowerSystem_LLM
+    python3 -m venv .venv
+    source .venv/bin/activate
+    ```
+  - Change the priority of the `statnett` mirroring repository from `supplemental` to `primary` using `nano` text editor:
+    ```commandline
+    nano pyproject.toml
+    ```
+  - Then:
+    ```commandline
+    poetry lock
```

Comment:
There is a mismatch in python versions as RNDP has been upgraded to Python 3.12.11 and the toml file has a strict requirement for 3.12.10. I changed the requirement to ^3.12.10 to run it (so any newer python 3.12 version will work)